linq.go

```diff
@@ -1218,6 +1219,42 @@ func (q Query) Average() (avg float64, err error) {
 	return
 }
 
+// Zip returns a query with the result of the "zipping" function
+// provided
+//
+// Example:
+// convertRoman:= func(n, m T) (T, error){
```

Comment:
Can you please indent the examples like the functions above and have proper gofmt'ing (like space before :=)